### PR TITLE
Added new headers for cache control to cameras snap endpoints

### PR DESCRIPTION
**Pull request by Majkluss**

### CODE REVIEW

File: prusa/link/web/cameras.py [hunk 1]
```diff
@@ -24,20 +29,41 @@ def photo_by_camera_id(camera_id):
...
                                     f"take a photo yet.")
+
+    last_modified_timestamp = int(time())
+    last_modified = datetime.utcfromtimestamp(last_modified_timestamp)
+    last_modified_str = last_modified.strftime(HEADER_DATETIME_FORMAT)
+
+    headers = {
+        'Last-Modified': last_modified_str,
+        'Date': last_modified_str,
+        'Expires': str(last_modified + timedelta(seconds=10)),
```
Comment (Majkluss) [inline prusa/link/web/cameras.py:40]: const

File: prusa/link/web/cameras.py [hunk 2]
```diff
@@ -24,20 +29,41 @@ def photo_by_camera_id(camera_id):
...
+        'Cache-Control': 'private, max-age=604800'
```
Comment (Majkluss) [inline prusa/link/web/cameras.py:41]: expire

File: prusa/link/web/cameras.py [hunk 3]
```diff
@@ -10,8 +12,11 @@
 from .lib.core import app
 from .lib.auth import check_api_digest
 
+# e.g. Mon, 07 Nov 2022 13:52:49 GMT
+HEADER_DATETIME_FORMAT = "%a, %d %b %Y %X GMT"
```
Comment (Majkluss) [inline prusa/link/web/cameras.py:16]: This value is used in more places, move it to the const

File: prusa/link/web/cameras.py [hunk 4]
```diff
@@ -26,20 +29,46 @@ def photo_by_camera_id(camera_id):
...
+    last_modified_str = last_modified.strftime(HEADER_DATETIME_FORMAT)
+
+    # Give PrusaLink some time to take a new snapshot
+    timeout = PHOTO_TIMEOUT + TIME_FOR_SNAPSHOT
+
+    expires = str(last_modified + timedelta(seconds=timeout))
+
+    headers = {
+        'Last-Modified': last_modified_str,
+        'Date': last_modified_str,
```
Comment (ondratu) [inline prusa/link/web/cameras.py:44]: I think, that `Date` header should be datetime of response....
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Date

**Review by ondratu — CHANGES_REQUESTED** [review]

**Review by ondratu — APPROVED** [review]

**Pull request merged**